tools/Az/Az.psd1

```diff
@@ -53,7 +53,7 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.1.2'; },
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.2.4'; },
```

Comment:
@abbhar we need this change in all the modules psd1 files. Lets modify all the psd1 files. we will find a better way next time